src/components/LiwoMap.vue

```diff
@@ -28,45 +37,74 @@
 <script>
 import 'leaflet/dist/leaflet.css'
 
+import Vue from 'vue'
 import L from 'leaflet'
-import { LMap, LTileLayer } from 'vue2-leaflet'
+import { LMap, LTileLayer, LGeoJson, LPopup } from 'vue2-leaflet'
 import 'proj4leaflet'
 
+import rijksdriehoek from '../lib/rijksdriehoek.js'
 import BaseLayerControl from './BaseLayerControl'
 import LiwoMapLayers from './LiwoMapLayers'
 
 import mapConfig from '../map.config'
 
+// as discussed in https://github.com/Leaflet/Leaflet/issues/4968
+// replace default icons by requires
+delete L.Icon.Default.prototype._getIconUrl
+L.Icon.Default.mergeOptions({
+  iconRetinaUrl: require('leaflet/dist/images/marker-icon-2x.png'),
+  iconUrl: require('leaflet/dist/images/marker-icon.png'),
+  shadowUrl: require('leaflet/dist/images/marker-shadow.png')
+})
+
 export default {
-  props: [ 'layerSet' ],
-  components: { BaseLayerControl, LiwoMapLayers, LMap, LTileLayer },
+  components: { BaseLayerControl, LMap, LiwoMapLayers, LTileLayer, LGeoJson, LPopup },
+  props: ['layerSet'],
   data () {
     return {
-      continuousWorld: true,
-      crs: this.createCrs(),
       zoom: mapConfig.zoom,
       maxZoom: mapConfig.maxZoom,
       minZoom: mapConfig.minZoom,
-      center: L.latLng(...mapConfig.center),
       attribution: mapConfig.attribution,
+      center: L.latLng(...mapConfig.center),
+      crs: this.createCrs(),
+      continuousWorld: true,
+      layers: [],
       baseLayer: {
         tms: mapConfig.tileLayers[0].tms,
         tileLayers: mapConfig.tileLayers,
         url: mapConfig.tileLayers[0].url
       }
     }
   },
+  mounted () {
+  },
   methods: {
+    onEachFeature (feature, layer) {
+      const parent = this.$refs[layer.id]
+      const Popup = Vue.extend(LPopup)
+      const popup = new Popup()
+      const result = popup.$mount()
+      console.log('result', result)
+      parent.mapObject.bindPopup(result.$el)
+    },
+
+    setStyle (feature, layer) {
+      // set the layer to to style object and use css for styling
+      return {className: layer.style}
+    },
     createCrs () {
-      return new L.Proj.CRS(mapConfig.crsType, mapConfig.proj, {
-        resolutions: mapConfig.resolutions,
-        bounds: L.bounds(mapConfig.bounds),
-        origin: mapConfig.origin
+      // Create the Coordinate Reference System
```

Comment:
moving this to a computed prop would solve this issue I think?

Reply:
I tried that, but it was caching, so returning the same instance multiple times.